Formula/wv2.rb

```diff
@@ -25,8 +25,12 @@ class Wv2 < Formula
   depends_on "glib"
   depends_on "libgsf"
 
+  uses_from_macos "libxml2"
+
   def install
-    ENV.append "LDFLAGS", "-liconv -lgobject-2.0" # work around broken detection
+    ENV.append "LDFLAGS", "-lgobject-2.0" # work around broken detection
+    ENV.append "LDFLAGS", "-liconv" if OS.mac?
+    ENV.append "CXXFLAGS", "-I#{Formula["libxml2"].include}/libxml2" unless OS.mac?
     system "cmake", ".", *std_cmake_args
     system "make", "install"
   end
```

Comment:
Can you add a test so we can actually validate this fix?

Reply:
I added a test now based on some upstream code, and it works in CI.